components/ui/label.tsx

```diff
@@ -7,7 +7,7 @@ import { VariantProps, cva } from "class-variance-authority"
 import { cn } from "@/lib/utils"
 
 const labelVariants = cva(
-  "text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+  "text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70 mb-3"
```

Comment:
This is a global scope component , changing here can cause a lot of ui bug in different part of site.

you can pass the additional styles as props. Avoid changing the design prototype

Reply:
also kindly note that styles works but the text is wrapped and last 4 digits are not shown cause it's overlaped with the icon 

<img width="584" alt="image" src="https://github.com/Abbhiishek/codecommunitymusic-client/assets/86338762/d56d68d0-90d7-41cf-86ab-ee903ceae7ba">

Have a look in the poicture you can too generate the issue

type : `1111111111111111111111111111111111111111111111111111111111222`

The last 2 digits are not visible. Kindly manage this scope